src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler.php

```diff
@@ -109,6 +109,34 @@ public static function register($mode = 0)
             }
 
             $trace = debug_backtrace(true);
+
+            // Silence deprecation warnings about private service accessed
+            // from the service container if done so from a Test class.
+            // As of Symfony 4.1, there is a new TestContainer that allows
+            // fetching of private services within tests, so we no longer
+            // need to warn about this behavior:
+            //
+            // https://symfony.com/blog/new-in-symfony-4-1-simpler-service-testing
```

Comment:
Should be removed, we usually don't link to the blog.

Reply:
Do you just want the link line removed (line 19 and surrounding spacing), or also line 16 and 17, which are highlighted above?